Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -237,8 +237,7 @@ def pull
         # Squash a Linuxbrew build-bottle-pr commit.
         if Utils.popen_read("git", "diff", orig_revision, "HEAD") =~ /^\+# .*: Build a bottle for Linuxbrew$/
           ohai "Squashing build-bottle-pr commit"
-          require "dev-cmd/squash-bottle-pr"
-          Homebrew.squash_bottle_pr
+          safe_system HOMEBREW_BREW_FILE, "squash-bottle-pr"
```

Comment:
Yeah, that had occurred to me, or autotap it. `brew test-bot` for example is autotapped when the command is first run. There's machinery for that. I prefer the warning message though. I'll first merge this bug-fix PR as is.